contrib/qutebrowser/template.py

```diff
@@ -0,0 +1,68 @@
+pkgname = "qutebrowser"
+pkgver = "3.1.0"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "asciidoc",
+    "gmake",
+    "python",
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-hypothesis",
+    "python-pytest",
+    "python-pytest-benchmark",
+    "python-pytest-mock",
+    "python-pytest-rerunfailures",
+]
+depends = [
+    "python-adblock",
+    "python-jinja2",
+    "python-pygments",
+    "python-pyqt6",
+    "python-pyqt6-webengine",
+    "python-pyqt6_sip",
+    "python-pyyaml",
+    "python-tldextract",
+    "qt6-qtbase",
+    "qt6-qtbase-dbus",
+    "qt6-qtbase-sql",
+    "qt6-qtwebengine",
+]
+pkgdesc = "Keyboard driven web browser with a minimalist gui"
+maintainer = "ttyyls <contact@behri.org>"
+license = "GPL-3.0-only"
+url = "https://qutebrowser.org"
+source = f"https://github.com/qutebrowser/qutebrowser/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "9307d55f1d9157b7906ac08566e3951bffd6f5b753432deaa9a13681995ba3ca"
+# unpackaged dependencies
+options = ["!check"]
+
+
+def post_build(self):
+    self.do("a2x", "-f", "manpage", "doc/qutebrowser.1.asciidoc")
+
+
+def post_install(self):
+    self.install_man("doc/qutebrowser.1")
+    self.install_dir("usr/share/qutebrowser")
+    self.install_files("scripts", "usr/share/qutebrowser")
```

Comment:
dont need the install_dir here p sure

Reply:
leftover from some previous glob install_file(s). I still don't think it's correct though since ideally I should also filter __pycache__ and test dirs and whatever like the makefile does here https://github.com/qutebrowser/qutebrowser/blob/main/misc/Makefile#L33 except to use the makefile i'd need to patch in an install -d "dir" for each since it relies on gnu behavior

Reply:
i would patch the makefile

Reply:
and remove the setup.py install from it too ofc

Reply:
you should also `make man` instead of manual a2x

Reply:
ok